package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "prepublish": "npm run build",
     "test": "npm run lint && npm run mocha",
-    "mocha": "mocha --require babel-core/register test/**/*.js",
+    "mocha": "mocha --require babel-core/register ./test/**/*.js ./test/*.js",
```

Comment:
Seems the glob pattern should be between single quotes: http://stackoverflow.com/a/34529436